lib/ts/recipe/session/recipe.ts

```diff
@@ -116,8 +118,12 @@ export default class Session extends RecipeModule<unknown, unknown, unknown, any
     private async getSessionContext({ action }: RecipeEvent): Promise<SessionContextType> {
```

Comment:
I think this can take a userContext in it since this is called in SessionAuth, and that should also take a userContext in it.

Reply:
This is still not done.

Reply:
Done